packages/keystatic/src/component-blocks/cloud-image-preview.tsx

```diff
@@ -10,65 +10,189 @@ import { Icon } from '@keystar/ui/icon';
 import { externalLinkIcon } from '@keystar/ui/icon/icons/externalLinkIcon';
 import { useConfig } from '../app/shell/context';
 
-export function CloudImagePreview(
-  props: PreviewProps<
-    ObjectField<typeof import('./cloud-image-schema').cloudImageSchema>
-  > & {
-    onRemove(): void;
-  }
-) {
-  const src = props.fields.src.value;
-  const alt = props.fields.alt.value;
+import { detect } from 'detect-browser';
 
-  const dimensions = useImageDimensions(src);
+const browser = detect();
+const metaSymbol = browser?.os === 'Mac OS' ? '⌘' : 'Ctrl';
+const isMobile = browser?.os === 'iOS' || browser?.os === 'Android OS';
```

Comment:
Since this is used for language around pointer events, a more accurate condition might be:

```js
import { useMediaQuery } from '@keystar/ui/style';

let isTouchDevice = useMediaQuery('(pointer: coarse)');
```